libraries/core-react/src/Select/NativeSelect/NativeSelect.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react'
+import { forwardRef, SelectHTMLAttributes } from 'react'
+import styled from 'styled-components'
+import { Label } from '../../Label'
+import { Select } from './Select'
+
+const Container = styled.div`
+  min-width: 100px;
+  width: 100%;
+`
+
+export type NativeSelectProps = {
+  /** Input unique id */
+  id: string
+  /** Label for the select element */
+  label: string
+  /** Meta text, for instance unit */
+  meta?: string
+  /** Input ref */
+  selectRef?: React.Ref<HTMLSelectElement>
+  /** Disabled state */
+  disabled?: boolean
+  /** The user can choose multiple items */
+  multiple?: boolean
+} & SelectHTMLAttributes<HTMLSelectElement>
+
+export const NativeSelect = forwardRef<HTMLDivElement, NativeSelectProps>(
+  function NativeSelect(
+    {
+      label,
+      children,
+      className,
+      selectRef,
+      id,
+      meta,
+      disabled = false,
+      multiple = false,
+      ...other
+    },
+    ref,
+  ) {
+    const containerProps = {
+      ref,
```

Comment:
Shouldn't the ref be passed to the `Select`component?

Reply:
To get i consistent I did it in the same way as the `<TextField>` component with one ref for the container and a separate ref for the select element 🤷‍♀️